assets/translations/en-us.yaml

```diff
@@ -1701,6 +1701,12 @@ typeDescription:
   resources.cattle.io.backup: A backup is created to perform one-time backups or schedule recurring backups based on a ResourceSet.
   resources.cattle.io.restore: A restore is created to trigger a restore to the cluster based on a backup file.
   resources.cattle.io.resourceset: A resource set defines which CRDs and resources to store in the backup.
+  monitoring.coreos.com.servicemonitor: A service monitor defines the group of services and the endpoints that Prometheus will scrape for metrics. This is the most common way to define metrics collection.
+  monitoring.coreos.com.podmonitor: A pod monitor defines the group of pods that Prometheus will scrape for metrics. The common way is to use service monitors, but pod monitors allow you to handle any situation where a service monitor wouldn't work.
+  monitoring.coreos.com.prometheusrule: A Prometheus rule defines groups of rules. There are two types of rules, recording rules can pre-compute values and save the results while alerting rules allow you to define conditions on when to send notifications to AlertManager.
```

Comment:
Rules needs some work...

A Prometheus Rule resource defines both recording and/or alert rules. A recording rule can pre-compute values and save the results. Alerting rules allow you to define conditions on when to send notifications to AlertManager.